mbq/client/client.py

```diff
@@ -11,12 +10,13 @@
 
 class ServiceClient:
     def __init__(self, api_url, auth=None, headers=None, post_process_response=None,
-                 default_timeout=30):
+                 get_correlation_id=None, default_timeout=30):
```

Comment:
This parameter name reads to me like it's expecting a boolean, consider something like `correlation_id_getter` or `get_correlation_id_fn` to make it sound more like a callable.

Reply:
Sorry that's Travis, ugh switching logins

Reply:
Also a docstring here talking about what this parameter does would be nice